src/engine/source/builder/definitions.cpp

```diff
@@ -61,7 +61,7 @@ void substituteDefinitions(Json& asset)
         if (!asset.erase(DEFINITIONS_KEY))
         {
             throw std::runtime_error(fmt::format(
-                "Engine definitions: Field \"{}\" from asset \"{}\" could not be erased.",
+                R"(Engine definitions: Field "{}" from asset "{}" could not be erased.)",
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C-plus-plus%29#modern-language-features-and-best-practices

Raw strings